frontend/apps/reader/modules/readergesture.lua

```diff
@@ -1300,6 +1304,10 @@ function ReaderGesture:gestureAction(action, ges)
         self.ui:handleEvent(Event:new("GotoNextChapter"))
     elseif action == "prev_chapter" then
         self.ui:handleEvent(Event:new("GotoPrevChapter"))
+    elseif action == "next_bookmark" then
+        self.ui:handleEvent(Event:new("GotoNextBookmarkFromPage", self.ui:getCurrentPage()))
+    elseif action == "prev_bookmark" then
+        self.ui:handleEvent(Event:new("GotoPreviousBookmarkFromPage", self.ui:getCurrentPage()))
```

Comment:
Still want to pass current page from there?

Reply:
I dont care either way, i am just keeping to the style in `readerbookmark` where all the functions are passed a page.

Reply:
I prefer as I proposed in https://github.com/koreader/koreader/pull/5968#discussion_r392711103 (no need to pass a page if the function/event is named ...FromCurrentPage)

Reply:
done